java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -623,9 +623,11 @@ private void startRaylet(String storeName, AddressInfo info, int numWorkers,
 
     String resourceArgument = ResourceUtil.getResourcesStringFromMap(staticResources);
 
+    int maximumStartupConcurrency = staticResources.get("CPU").intValue();
```

Comment:
Similar to `psutil.cpu_count()` in Python, Java has `Runtime.getRuntime().availableProcessors()`

Reply:
I added that in to match the Python side.